src/components/JitsiMeeting.tsx

```diff
@@ -0,0 +1,126 @@
+import { useCallback, useEffect, useRef, useState } from 'react';
+
+import { initExternalApi } from '..';
+import { IJitsiMeetingProps } from '../types';
+import { generateComponentId } from '../utils';
+
+/**
+ * Returns the JitsiMeeting Component with access to a custom External API
+ * to be used as-it-is in React projects
+ *
+ * @param {IJitsiMeetingProps} props the component's props
+ * @returns {React.Component} the `JitsiMeeting` Component
+ * @example
+  ```js
+    <JitsiMeeting
+        domain='meet.jit.si'
+        roomName: 'TestingJitsiMeetingComponent'
+        width: '100%'
+        height: 500
+        spinner={CustomSpinner}
+        onApiReady={(externalApi) => console.log(externalApi)}
+    />
+  ```
+ */
+const JitsiMeeting = ({
+    domain,
+    roomName,
+    width,
+    height,
+    configOverwrite,
+    interfaceConfigOverwrite,
+    jwt,
+    onload,
```

Comment:
I wonder if we want to expose the `onload` handler...

Reply:
Not sure, what reasons are there not to? Maybe some integrators could use it to load extra functionality.

Reply:
It seems to overlap with the spinner. That exists so users know when to display some loading indication. But id we provide it I don't see its usefulness.

As a general note, it's always easier to extend an API than to take things back ;-)

Reply:
Fair enough 😃